mmv1/third_party/terraform/services/composer/resource_composer_environment.go.erb

```diff
@@ -237,17 +243,37 @@ func ResourceComposerEnvironment() *schema.Resource {
 										Type:             schema.TypeString,
 										Computed:         true,
 										Optional:         true,
+<% if version == "ga" -%>
 										ForceNew:         true,
+<% else -%>
+										ForceNew:         false,
+										ConflictsWith:    []string{"config.0.node_config.0.composer_network_attachment"},
+<% end -%>
 										DiffSuppressFunc: tpgresource.CompareSelfLinkOrResourceName,
 										Description:      `The Compute Engine machine type used for cluster instances, specified as a name or relative resource name. For example: "projects/{project}/zones/{zone}/machineTypes/{machineType}". Must belong to the enclosing environment's project and region/zone. The network must belong to the environment's project. If unspecified, the "default" network ID in the environment's project is used. If a Custom Subnet Network is provided, subnetwork must also be provided.`,
 									},
 									"subnetwork": {
 										Type:             schema.TypeString,
 										Optional:         true,
+<% if version == "ga" -%>
 										ForceNew:         true,
+<% else -%>
+										ForceNew:         false,
+										Computed:         true,
+										ConflictsWith:    []string{"config.0.node_config.0.composer_network_attachment"},
+<% end -%>
 										DiffSuppressFunc: tpgresource.CompareSelfLinkOrResourceName,
-										Description:      `The Compute Engine subnetwork to be used for machine communications, , specified as a self-link, relative resource name (e.g. "projects/{project}/regions/{region}/subnetworks/{subnetwork}"), or by name. If subnetwork is provided, network must also be provided and the subnetwork must belong to the enclosing environment's project and region.`,
+										Description:      `The Compute Engine subnetwork to be used for machine communications, specified as a self-link, relative resource name (e.g. "projects/{project}/regions/{region}/subnetworks/{subnetwork}"), or by name. If subnetwork is provided, network must also be provided and the subnetwork must belong to the enclosing environment's project and region.`,
 									},
+<% unless version == "ga" -%>
+									"composer_network_attachment": {
```

Comment:
Oh, we'll need to add handwritten documentation for the field: https://googlecloudplatform.github.io/magic-modules/develop/resource/#add-documentation

Reply:
I have a separate [PR](https://github.com/GoogleCloudPlatform/magic-modules/pull/10012) for documenting Composer 3 argument usage.